#latitude is in degree
float64 latitude
#longitude is in degree
float64 longitude
float32 altitude
#yaw is in degree from -180 to 180
float32 yaw
---
bool result 
